### DISCUSSION TITLE
Research/plan features for v1.X+

### DISCUSSION THREAD

**Discussion by otech47** [setlife-network/trinary#579] [Ideas]
# Notes

- Who would use this kind of tool?
1. Small business/freelance owners/managers with more than a few projects that they need to track performance/expenses for
2. Managers inside larger corporations/companies that may want to measure performance across different teams (performance reviews, bonuses, commissions calculation)
3. Nonprofit sector for justifying funding programs 

- longer-term projects?
- forecasting from 6+ months of data, (conservative-aggressive ranges of scenarios)
- Releases are groups of issues that closed in a certain time period and is strongly associated with customer satisfaction and making more payments

**Comment by mecoccaro** [comment]
@otech47 How could we tackle this issue? Or what kind of information we can obtain of this in order to improve a release or a milestone??

**Comment by otech47** [comment]
@mecoccaro This should be more of a discussion but the issue was created before GitHub Discussions existed!
